roles/kubernetes-apps/helm/defaults/main.yml

```diff
@@ -10,6 +10,24 @@ helm_deployment_type: host
 # Do not download the local repository cache on helm init
 helm_skip_refresh: false
 
+# Options to secure Tiller installation
+tiller_enable_tls: false # Use to the conditionally generate TLS certs
+
+# Store tiller release information as Secret instead of a ConfigMap
+tiller_secure_release_info: false
+helm_config_dir: "{{ kube_config_dir }}/helm"
+helm_script_dir: "{{ bin_dir }}/helm-scripts"
+
```

Comment:
`helm_home_dir` is not defined

Reply:
helm_home_dir is already defined on line 5 (preexisting variable)